### PR TITLE
Add did:elem

### PR DESCRIPTION
**Pull request by OR13**
Elem is an independent implementation of the sidetree protocol used by `did:ion`, https://github.com/decentralized-identity/sidetree/blob/master/docs/protocol.md

Elem relies on Ethereum instead of Bitcoin.

### CODE REVIEW

**Review by msporny — DISMISSED** [review]
The linked DID Method spec doesn't contain the sections required by the DID Core specification. See this comment for more information: https://github.com/w3c-ccg/did-method-registry/pull/75#issuecomment-575461028

**Comment by gjgd** [comment]
@msporny Solved the conflict and updated the README to contain the did method specification containing the required documentation (did method format + CRUD operations + security considerations). Could you review again?

**Comment by gjgd** [comment]
fixes https://github.com/decentralized-identity/element/issues/142

**Comment by gjgd** [comment]
ping @msporny when you get a chance to review ^

File: index.html [hunk 1]
```diff
@@ -965,6 +965,23 @@ <h1>The Registry</h1>
...
+        PROVISIONAL
+      </td>
+      <td>
+        Element DID
+      </td>
+      <td>
+        Transmute
+      </td>
+      <td>
+        <a href="https://github.com/decentralized-identity/element">ELEM DID Method</a>
```
Comment (msporny) [inline index.html]:
```suggestion
        <a href="https://github.com/decentralized-identity/element/blob/master/docs/did-method-spec/spec.md">ELEM DID Method</a>
```

**Review by msporny — CHANGES_REQUESTED** [review]
Please link directly to the DID Method spec. Other than that, LGTM for inclusion into the registry. The documentation is weak and would probably not enable anyone else to implement the method in an interoperable fashion.

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html]: Fix remaining merge conflicts too, please.

**Comment by gjgd** [comment]
@msporny fixed the link and the conflict

**Review by msporny — APPROVED** [review]

**Pull request merged**